File: benchmarks/js/fib_rec.js
// Benchmark: Fibonacci (recursive)
function fib(n) {
    if (n < 2) return n;
    return fib(n - 1) + fib(n - 2);
}
let total = 0;
for (let i = 0; i < 5; i++) {
    total += fib(28);
}
console.log(total);

File: benchmarks/py/fib_rec.py
# Benchmark: Fibonacci (recursive)
import sys
sys.setrecursionlimit(10000)

def fib(n):
    if n < 2: return n
    return fib(n - 1) + fib(n - 2)

total = 0
for i in range(5):
    total += fib(28)
print(total)
